readme.md

```diff
@@ -11,6 +11,12 @@ This is for advanced users. You probably want to use XO directly.
 $ npm install --save-dev eslint-config-xo
 ```
 
+For the `esnext` version you'll also need Babel's eslint parser and plugin:
+
+```
+$ npm install --save-dev babel-eslint eslint-plugin-babel
+```
```

Comment:
Yes, that's what I'm thinking too, but still doesn't solve it for people using this directly...